doc/user-guide.adoc

```diff
@@ -215,3 +215,101 @@ spec:
+For that purpose the WildFly operator provides scale down functionality which verifies if all transactions were finished
+and only then marks the pod as clean for termination.
+
+Decreasing the replica size in the `WildFlyServer` customer resource is done at field `WildFlyServer.Spec.Replicas` (see <<size>>).
+You can use for example patch command like
+
+```
+oc patch wildflyserver <<name>> -p '[{"op":"replace", "path":"/spec/replicas", "value":0}]' --type json
+```
+
+or you can manually edit and change the replica number with with `oc edit wildflyserver <<name>>`.
+
+NOTE: Decreasing replica size at the `StatefulSet` or deleting the Pod itself has no effect and as such changes will be reverted.
+
+WARNING: if you decide to delete whole `WildflyServer` definition (`oc delete wildflyserver <<deployment_name>>`)
+         then no transaction recovery process is started and the pod is terminated regardless of unfinished transactions.
+         If you want to remove the deployment and you expects doing it in transaction safe way
+         you need first to scale down the number of pods to 0,
+         wait until all pods are terminated
+         and only after that you can delete the `WildFlyServer` instance
+
+WARNING: Narayana recovery listener has to be enabled in Wildfly transaction subsystem.
+         Otherwise scaledown transaction recovery processing is skipped for the particular WildFly pod.
+         See the `recovery-listener` at
+         https://wildscribe.github.io/WildFly/18.0/subsystem/transactions/index.html.
+         The recovery listener is **not** enabled by default!
+
+when the scaledown process begins the pod state (`oc get pod <<pod_name>>``) is still marked as `Running`.
+The reason is that that the pod needs to be able to finish all the unfinished transactions and which includes the remote EJB calls that targets it.
+If you want to observe the state of the scaledown processing you need to observe the status of the `WildFlyServer` instance.
+When running `oc describe wildflyserver<<name>>` you can see the status of the Pods.
+
+The `WildFlyServer.Status.Pods[].State` can be on of these
+
+|===
+| Status.Pod.State | Description
+
+| ACTIVE
+| The pod is active and processing requests.
+
+| SCALING_DOWN_RECOVERY_INVESTIGATION
+| The pod is about to be scaled down. The scale down process is under investigation about state of transactions in WildFly.
+
+| SCALING_DOWN_RECOVERY_DIRTY
+| The WildFly contain some unfinished transactions. The pod cannot be terminated until they are cleaned.
+  The transaction recovery is periodically run at WildFly and it waits the transactions are finished eventually.
+
+| SCALING_DOWN_CLEAN
+| The pod was processed by transaction scaled down processing and is marked as clean to be removed from the cluster.
+
+|===
+
+
+You can observe the overall state of the active and no-active pods when take a look at the
```

Comment:
"by looking at the `WildFlyServer.Status.ScalingdownPods` and `WildFlyServer.Status.Replicas` fields"